apps/backend/src/modules/auth/auth.service.ts

```diff
@@ -1,6 +1,6 @@
 import { ExceptionMessage, HTTPCode } from "~/libs/enums/enums.js";
 import { Encrypt } from "~/libs/modules/encrypt/encrypt.js";
-import { Token } from "~/libs/modules/token/token.js";
+import { type Token } from "~/libs/modules/token/token.js";
```

Comment:
```suggestion
import { Token } from "~/libs/modules/token/token.js";
```
do not add any unnecessary changes